runtime/src/accounts_background_service.rs

```diff
@@ -629,26 +629,38 @@ impl AccountsBackgroundService {
                                     break;
                                 }
                             }
-                        } else if previous_clean_time.elapsed() > CLEAN_INTERVAL {
-                            // Note that the flush will do an internal clean of the
-                            // cache up to bank.slot(), so should be safe as long
-                            // as any later snapshots that are taken are of
-                            // slots >= bank.slot()
-                            bank.force_flush_accounts_cache();
-                            bank.clean_accounts();
-                            last_cleaned_slot = bank.slot();
-                            previous_clean_time = Instant::now();
-                            bank.shrink_ancient_slots();
-                            bank.shrink_candidate_slots();
-                            previous_shrink_time = Instant::now();
                         } else {
-                            // Note that the flush will do an internal clean of the
-                            // cache up to bank.slot(), so should be safe as long
-                            // as any later snapshots that are taken are of
-                            // slots >= bank.slot()
-                            bank.flush_accounts_cache_if_needed();
```

Comment:
I think this pr removes the last caller anywhere of this.
https://github.com/anza-xyz/agave/blob/347b25d7bf159a7074b9d9f0db83c58cacdd6853/runtime/src/bank.rs#L4831